assets/css/filtersection.css

```diff
@@ -55,7 +38,353 @@
     background-color: var(--verylightgray);
 }
 
-/* Show the dropdown menu (use JS to add this class to the .dropdown-content container when the user clicks on the dropdown button) */
-.show {
+#filters .filter-links {
+    position: relative;
+    margin-bottom: 30px; 
+}
+
+.filter-links.size-toggle {
+    position: relative;
+    vertical-align: top;
+    padding: 0;
+}
+
+.filter-links.size-toggle a {
+    position: relative;
+}
+
+.filter-links a span {
```

Comment:
Again, I highly recommend simplifying all your CSS selectors to single classes unless you absolutely need the extra specificity. For the most part, you won't.

Further, it's generally bad practice to include HTML tags in selectors, because your CSS will break if you change your DOM structure.